kubernetes/kfserving/kf_request_json/totensor.py

```diff
@@ -0,0 +1,25 @@
+import torch
+from PIL import Image
+from torchvision import transforms
+image = Image.open('0.png')  # PIL's JpegImageFile format (size=(W,H))
+print(image.size)  # (W，H）
```

Comment:
What about text inputs? Is there a conversion function needed?

Reply:
Added a script to generate input json for both image and text input.